tests/output/test_add_instance.json

```diff
@@ -0,0 +1,13 @@
+{
+  "7bad2efc-cb40-420b-aef9-2f909d038b46":   {
+    "meta": "http://onto-ns.com/meta/0.1/Result",
+    "dimensions": {
+      "natoms": 2,
+      "ncoords": 3
+    },
+    "properties": {
+      "potential_energy": 0,
+      "forces": [[0, 0, 0], [0, 0, 0]]
+    }
+  }
+}
```

Comment:
There is no point in committing a file that is generated.

But more worrying is that the properties are uninitialised. We should test that they are correctly assigned.